Register and VAM Adjustments

biellamariscotes opened the pull request:
### Overview
- This PR refactors the VAM to the current flow. Also adds a note in the Register.

### Details
- Adds Student ID and full name details in Voter Details and Validate Voter:
![image](https://github.com/user-attachments/assets/62645ec5-0118-4743-8dc7-c35f1bfcbfee)
![image](https://github.com/user-attachments/assets/1bfb7a1b-ecda-4285-9d0c-e467e943f246)

- Refactors the table, replaced email with full names. Also implemented validation in the searching.

https://github.com/user-attachments/assets/dbaf8bf5-3ca0-4863-87ae-7c25b93a4599

biellamariscotes committed:
style: added note regarding the disabled course options

biellamariscotes committed:
fix: highlight of the pages on the admin-side bar

biellamariscotes committed:
style: removed cor and adjusted the elements

biellamariscotes committed:
fix: regression bug on the opening of modals

biellamariscotes committed:
refactor: changed email to full name

biellamariscotes committed:
feat: validation on searching

biellamariscotes committed:
Merge branch 'development' into Register-VAM

Carl-Tabuso commented on src/manage-voters.php:
Do change the jquery version to 3.7.1 so we won't have conflicting versions with other page files.

```html
<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```

Also, add a defer attribute in the script tag of loader script.

```html
<script src="scripts/loader.js"></script>
```

or just simply move the script within the body—where all your external scripts are placed.

To make sure that the preloader is prioritized before your page renders, you can use the `preload` value for `rel` attribute. Kind of similar to this implementation.

```html
  <!-- Preloader Stylesheet and Image -->
  <link rel="preload" href="images/resc/ivote-icon.webp" as="image">
  <link rel="preload" href="styles/loader.css" as="style" />
  <link rel="stylesheet" href="styles/loader.css" />
```

biellamariscotes replied:
Settled in commit [8611c64](https://github.com/BSIT-3-1-APPDEV/PUPSRC-AutomatedElectionSystem/pull/299/commits/8611c6492da59715d7ea1a1d450fafc9847a90d0).

Carl-Tabuso reviewed (changes requested):
### Cosmetics

- Can you place the search icon inside the search input bar.
- You forgot to change the table header from Email Address to Full Name(also applicable to Voters' Accounts table).

![image](https://github.com/user-attachments/assets/112a0f46-eaf9-455a-b3b5-bccaf93c5c59)

- Would also like to add that you should actually create a student number column, ideally the first column. This shall serve as the unique identifier for each student as per sir chris and ma'am emy.

biellamariscotes committed:
Merge branch 'development' into Register-VAM

biellamariscotes committed:
style: changed table th name and search icon position

biellamariscotes committed:
style: added student id

biellamariscotes commented:
> ### Cosmetics
> * Can you place the search icon inside the search input bar.
> * You forgot to change the table header from Email Address to Full Name(also applicable to Voters' Accounts table).
> 
> ![image](https://private-user-images.githubusercontent.com/162773681/353303378-112a0f46-eaf9-455a-b3b5-bccaf93c5c59.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI1MTgwNjgsIm5iZiI6MTcyMjUxNzc2OCwicGF0aCI6Ii8xNjI3NzM2ODEvMzUzMzAzMzc4LTExMmEwZjQ2LWVhZjktNDU1YS1iM2I1LWJjY2FmOTNjNWM1OS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgwMVQxMzA5MjhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kN2NhZTYxMTQyMWVhOGFhYmZjYTMzNWFjOWEwMDEwNWI1ODgzNDk5MjU2ODFmY2ZlMjgxODFmOGE2NDM2MDM2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.U5u5gcK28NocN4-nUsf8gvxz1_kRhbwa_HAezh59fJs)
> 
> * Would also like to add that you should actually create a student number column, ideally the first column. This shall serve as the unique identifier for each student as per sir chris and ma'am emy.

![image](https://github.com/user-attachments/assets/e65b321d-55ba-47ce-978e-d95bdb1aab7b)

I have implemented the requested changes. The full name remains the first column since it serves as a hyperlink to the student's information. Using the full name is more appropriate for this redirection than the student ID, in my opinion.

biellamariscotes committed:
chore: update jquery version to 3.7.1 and optimize script loading

biellamariscotes committed:
Merge branch 'development' into Register-VAM

biellamariscotes committed:
refactor: removed add voters and hide pending registration from SCO

biellamariscotes commented:
COMMIT [804e231](https://github.com/BSIT-3-1-APPDEV/PUPSRC-AutomatedElectionSystem/pull/299/commits/804e231400e92c788431b38d9e01faa2d65d7e64)

Also removed the Add Voters, as well as the hiding of the pending registrations in SCO.

Excuse the data in the table, it's not updated.

![image](https://github.com/user-attachments/assets/54e16044-d777-42ed-9d34-d518b4216f6e)

biellamariscotes committed:
style: adjusted the vector placement

Carl-Tabuso commented on src/manage-voters.php:
```diff
@@ -67,166 +68,168 @@ class="hide-text">MANAGE USERS</span>
+															</thead>
+															<tbody>
+																<?php while ($row = $to_verify_tbl->fetch_assoc()) { ?>
+																	<!-- Generated in table-funcs.js -->
```
This is what "Pending Registration" table looks like on my end.

![image](https://github.com/user-attachments/assets/9f2e01d0-e3b4-407d-a016-b444a3ae7d89)

biellamariscotes committed:
fix: added the missing student id

biellamariscotes committed:
Merge branch 'development' into Register-VAM

biellamariscotes commented:
@Carl-Tabuso fixed na

biellamariscotes committed:
fix: reverted search style

biellamariscotes committed:
refactor: reverted back to 3.7.1

Carl-Tabuso reviewed (commented):
You seem to have forgotten the Email Address header. Change this to Full Name and then we proceed with the merging.

![image](https://github.com/user-attachments/assets/49858c71-9689-4f9e-856f-50eaf86eed39)

biellamariscotes committed:
fix: table header moving when searching

biellamariscotes commented:
> > ### Cosmetics
> > 
> > * Can you place the search icon inside the search input bar.
> > * You forgot to change the table header from Email Address to Full Name(also applicable to Voters' Accounts table).
> > 
> > ![image](https://private-user-images.githubusercontent.com/162773681/353303378-112a0f46-eaf9-455a-b3b5-bccaf93c5c59.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI1MTgwNjgsIm5iZiI6MTcyMjUxNzc2OCwicGF0aCI6Ii8xNjI3NzM2ODEvMzUzMzAzMzc4LTExMmEwZjQ2LWVhZjktNDU1YS1iM2I1LWJjY2FmOTNjNWM1OS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgwMVQxMzA5MjhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kN2NhZTYxMTQyMWVhOGFhYmZjYTMzNWFjOWEwMDEwNWI1ODgzNDk5MjU2ODFmY2ZlMjgxODFmOGE2NDM2MDM2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.U5u5gcK28NocN4-nUsf8gvxz1_kRhbwa_HAezh59fJs)
> > 
> > * Would also like to add that you should actually create a student number column, ideally the first column. This shall serve as the unique identifier for each student as per sir chris and ma'am emy.
> 
> ![image](https://private-user-images.githubusercontent.com/135019439/354247477-e65b321d-55ba-47ce-978e-d95bdb1aab7b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI1MjI5MDgsIm5iZiI6MTcyMjUyMjYwOCwicGF0aCI6Ii8xMzUwMTk0MzkvMzU0MjQ3NDc3LWU2NWIzMjFkLTU1YmEtNDdjZS05NzhlLWQ5NWJkYjFhYWI3Yi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgwMVQxNDMwMDhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hZGY5ODlkNjQ2MzAwN2ZlYzMxMzllMTAzMjI2MzA0MjNkMzVlMjM3NzdiODE5OTZkYTdjNjYwYTQ1NjQ1MjI2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.Z8TbwmuhAVRncu4plp9N5gWFQOfTA8QI8ISyW-bQets)
> 
> I have implemented the requested changes. The full name remains the first column since it serves as a hyperlink to the student's information. Using the full name is more appropriate for this redirection than the student ID, in my opinion.

![image](https://github.com/user-attachments/assets/96cdd476-6c8d-4722-9925-0c26cc62832b)

Reverted back the missing codes that weren't included upon commit.
